tests/inference/Array.java

```diff
@@ -3,6 +3,7 @@
 public class Array {
 
 	int @Rep[] pi;
```

Comment:
Check indentation

Reply:
Isn't the indentation on line 5 still off?

Reply:
There was an issue with tabs vs spaces so it was looking fine on my IDE. Fixed that.